### PR TITLE
New custom theme added

### PR DESCRIPTION
**Pull request by AlexanderVTr**

### CODE REVIEW

**Comment by AlexanderVTr** [comment]
- to the main file Root.tsx added new component CustomTheme
- CustomTheme.js contain styles that overwrite current theme
- added new logo and background images to Custom theme

File: packages/apps/src/Root.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ import { ApiStatsCtxRoot, BlockAuthorsCtxRoot, BlockEventsCtxRoot, KeyringCtxRoo
 import { settings } from '@polkadot/ui-settings';
 
 import Apps from './Apps.js';
+import { CustomTheme } from './customTheme.js';
```
Comment (dedok) [inline packages/apps/src/Root.tsx:16]: Can we rename it to QNFTheme.js? if it possible, then let's rename files as well.

**Review by dedok — COMMENTED** [review]
I address some questions / ideas. Pls take a look and let me know that you think.

File: packages/apps/src/Root.tsx (hunk 1 above)
Comment (AlexanderVTr) [reply packages/apps/src/Root.tsx]: Fixed

**Pull request merged**